oxidation/libparsec/crates/types/src/id.rs

```diff
@@ -332,7 +332,7 @@ impl HumanHandle {
             || label.chars().any(|c| {
                 matches!(
                     c,
-                    '(' | ')' | '<' | '>' | '@' | ',' | ':' | ';' | '\\' | '"' | '[' | ']'
+                    '<' | '>' | '@' | ',' | ':' | ';' | '\\' | '"' | '[' | ']'
```

Comment:
```suggestion
                    '<' | '>' | '@' | ',' | ':' | ';' | '"' | '[' | ']'
```

We still allow backslash ?